application/models/post_fixtures_test.go

```diff
@@ -177,15 +177,15 @@ func createFixturesForPostFindByUserAndUUID(ms *ModelSuite) PostFixtures {
 	}
 }
 
-//        Org0                Org1          Org2
-//        |  |                |  |           |
-//        |  +----+-----------+  +-----+-----+
-//        |       |                    |
-//       User1  User0                Trust
+//        Org0                Org1           Org2
+//        |  |                | | |          | |
+//        |  +----+-----------+ | +----+-----+ +
+//        |       |             |      |       |
+//       User1  User0        User3   Trust   User2
 //
-// Org0: Post0, Post2, Post3, Post4
-// Org1: Post1
-// Org2: Post5, Post6, Post7
+// Org0: Post0 (SAME), Post2 (SAME), Post3 (SAME), Post4 (SAME)
```

Comment:
It might be worth including the post status in the parenthesis or something that indicates whether it should be filtered out of the list.

Reply:
Good suggestion.